sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -106,9 +106,9 @@ namespace Azure { namespace Storage { namespace Blobs {
     std::string Key;
 
     /**
-     * @brief Base64 encoded string of the AES256 encryption key's SHA256 hash.
+     * @brief SHA256 hash of the AES256 encryption key.
      */
-    std::string KeyHash;
```

Comment:
This change is not mentioned in change log.

Reply:
I was thinking, shouldn't this also be `ContentHash`, with the HashAlgorithm bing `Sha256`?